app/Helpers/database/player-game.php

```diff
@@ -258,6 +258,7 @@ function getUserAchievementUnlocksForGame(User|string $user, int $gameID, Achiev
         ->join('game_achievement_sets', 'game_achievement_sets.achievement_set_id', '=', 'achievement_sets.id')
         ->where('game_achievement_sets.game_id', $gameID)
         ->where('Flags', $flag->value)
+        ->orderBy('player_achievements.achievement_id')
```

Comment:
Adding the constraint seems to have modified the SQL execution plan for this query (ultimately causing a test to fail). I've added this `orderBy` to fix the test and make the returned achievement IDs deterministic.